src/Mod/Path/PathScripts/post/rrf_post.py

```diff
@@ -0,0 +1,775 @@
+# -*- coding: utf-8 -*-
+# *****************************************************************************
+# *                                                                           *
+# *   (c) sliptonic (shopinthewoods@gmail.com) 2014                           *
+# *   (c) Gauthier Briere - 2018, 2019                                        *
+# *   (c) Schildkroet - 2019-2020                                             *
+# *   (c) Gary L Hasson - 2020                                                *
```

Comment:
Format for header is:
`# *   Copyright (c) YEAR FirstLast/Nickname <foo@bar.net>              *`

So it would be:
```
# *  Copyright (c) 2014 sliptonic <shopinthewoods@gmail.com>                            *
# *  Copyright (c) 2018-2019 Gauthier Briere                                          *
# *  Copyright (c) 2019-2020 Schildkroet                                              *
# *  Copyright (c) 2020 Gary L Hasson                                                 *
```

Reply:
I've made a PR in your upstream repo, @P-C-R  
https://github.com/P-C-R/FreeCAD-1/pull/1

Reply:
Thanx!